src/pages/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -6,192 +6,90 @@ import { Callout } from "/src/components/callout"
 The Axelar Virtual Machine (AVM) and Amplifier are currently under active development, so these instructions are likely to change. Please check back frequently for updates.
 </Callout>
 
-By running a **verifier** (also called a **worker**) for a chain integration with Axelar, you will be responsible for voting on the validity of transactions on the Axelar network for a given chain.
+By running a **verifier** (also called a **worker**) for a chain integration with Axelar, you will be responsible for voting on the validity of transactions on the Axelar network and signing command batches for a given chain.
 
 ## Prerequisites
 
```

Comment:
do we need both `tofnd` and `ampd`? we only show instructions on how to download `ampd` - are the terms used interchangeably in this case? @cjcobb23

Reply:
you need tofnd and ampd

Reply:
ok not sure why those were deleted. will add it back